### DISCUSSION TITLE
Oracle String NCLOB

### DISCUSSION THREAD

**Discussion by awahbahplana** [elsa-workflows/elsa-core#2610] [Q&A]
I have Problem with Oracle all string columns are NVARCHAR2(2000) and I need to use it as NCLOB 
I saw the problem are solved but i think in next version 2.4 when it release or prerelease?

**Comment by sfmskywalker** [comment]
You can try the 2.4 preview packages today from MyGet to see if it solves your issue.
The 2.4 stable release should happen some time this week.
